.github/renovate.json5

```
@@ -37,6 +37,18 @@
       packageNameTemplate: 'rust-lang/rust',
       datasourceTemplate: 'github-releases',
     },
+    {
+      customType: 'regex',
+      fileMatch: [
+        '^.github.workflows.main.yml$',
+      ],
+      matchStrings: [
+        'cargo-semver-checks.releases.download.v(?<currentValue>\\d+\\.\\d+(\\.\\d+)?)',
```

Slashes should be fine here?

```suggestion
        'cargo-semver-checks/releases/download/v(?<currentValue>\\d+\\.\\d+(\\.\\d+)?)',
```

---

Are they?  In some regexes, `/` are separators and with how hard it is to test and debug RenovateBot, I was taking a more cautious route

---

Given it is a Node.js project, agree you are right being cautious.